dev/scripts/src/alluxio.org/build/artifact/artifact-methods.go

```diff
@@ -22,25 +22,33 @@ import (
 	"alluxio.org/common/command"
 )
 
-func NewArtifact(artifactType ArtifactType, outputDir, targetName, version string, metadata map[string]string) (*Artifact, error) {
+func NewArtifactGroup(version string) (*ArtifactGroup, error) {
 	hOut, err := command.New("git rev-parse --short HEAD").Output()
 	if err != nil {
 		return nil, stacktrace.Propagate(err, "error getting commit hash")
 	}
 
-	return &Artifact{
-		Type:     artifactType,
-		Path:     filepath.Join(outputDir, targetName),
-		Version:  version,
-		Metadata: metadata,
+	return &ArtifactGroup{
+		Artifacts: []*Artifact{},
```

Comment:
this line shouldn't be necessary if you're going to be appending to it. `arr := append(nil, a)` works

Reply:
updated